### PR TITLE
- Redesign and prioritize the color enhancement and homework

### PR DESCRIPTION
**Pull request by dondomiedungca**

### CODE REVIEW

File: src/components/dropdown.js [hunk 1]
```diff
@@ -1,56 +1,62 @@
-import React from 'react';
-import { View, StyleSheet, Text } from 'react-native';
-import { SelectList } from 'react-native-dropdown-select-list';
-import checkInput from './inputChecker';
+import React from "react";
```
Comment (B0ER) [inline src/components/dropdown.js]: has it done by prettier?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "web": "expo start --web"
   },
   "dependencies": {
+    "@expo/vector-icons": "^14.0.4",
```
Comment (B0ER) [inline package.json:15]: don't you know how react applications build? (I need information here)
does it includ all dependencies to a build, or automatically link deps by imports, or manually link by expo/bable builder?

**Comment by MenlyCSE** [comment]
@dondomiedungca Thanks for the pull request! Before I merge and ultimately test things, can you please provide a description? It would be helpful to know whether some of these files affect functionality or simply makes stylistic changes to the user interface.

File: src/components/dropdown.js (hunk 1 above)
Comment (B0ER) [reply src/components/dropdown.js]: (Added prettier to the project, please make rebase for your branch)

File: package.json (hunk 2 above)
Comment (B0ER) [reply package.json:15]: Upd: seems It doesn't important whatever section of deps will be for react-native project, as there babel exists

**Comment by dondomiedungca** [comment]
> @dondomiedungca Thanks for the pull request! Before I merge and ultimately test things, can you please provide a description? It would be helpful to know whether some of these files affect functionality or simply makes stylistic changes to the user interface.

File: src/components/dropdown.js (hunk 1 above)
Comment (dondomiedungca) [reply src/components/dropdown.js]: Yes, this was caused by prettier, anyways I'm done in rebase, I updated the PR now. Thanks

File: package.json (hunk 2 above)
Comment (dondomiedungca) [reply package.json:15]: I used the vector icons for the homework screen and also for the custom checkbox, looks like it doesn't affect the build process.

**Comment by dondomiedungca** [comment]
> @dondomiedungca Thanks for the pull request! Before I merge and ultimately test things, can you please provide a description? It would be helpful to know whether some of these files affect functionality or simply makes stylistic changes to the user interface.

**Comment by dondomiedungca** [comment]
Hi @MenlyCSE here are the changes for this PR

- Redesign and prioritize the color enhancement and homework
- Input field focus effect
- Update border thickness
- Color combination especially on the profile button
- Install vector icons for checkbox and homework icons

**Review by MenlyCSE — APPROVED** [review]
Job well done! These changes don't conflict with our existing changes so I'll merge it later to try things. Or feel free to merge these changes now

**Pull request merged**